Fix for "undefined symbol: cblas_..." with USE="benchmark" and FEATURES="test"
--- a/clients/CMakeLists.txt
+++ b/clients/CMakeLists.txt
@@ -177,8 +177,9 @@ if( BUILD_CLIENTS_BENCHMARKS OR BUILD_CLIENTS_TESTS)
         endif()
       endif()
     else()
-      find_package( BLAS REQUIRED )
-      set( BLAS_LIBRARY "${BLAS_LIBRARIES}" )
+      find_package( CBLAS CONFIG REQUIRED )
+      set( BLAS_LIBRARY "${CBLAS_LIBRARIES}" )
+      set( BLAS_INCLUDE_DIR "${CBLAS_INCLUDE_DIRS}" )
     endif()
   else() # WIN32
     file(TO_CMAKE_PATH "C:/Program\ Files/AMD/AOCL-Windows/amd-blis/lib/ILP64/AOCL-LibBlis-Win-MT.lib" AOCL_BLAS_LIBRARY)
